src/middlewared/middlewared/plugins/update_/freebsd_to_scale_linux.py

```diff
@@ -9,14 +9,20 @@
 
 
 class UpdateService(Service):
+
+    @private
+    def remove_files(self):
+        with contextlib.suppress(FileNotFoundError):
```

Comment:
Context manager needs to be applied to each remove operation independently; otherwise, non-existence of one file will prevent the removal of subsequent ones. Also, IIRC, in the pathlib library, there is a flag to remove a file a be ok if it does not exist.